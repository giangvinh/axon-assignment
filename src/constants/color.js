export const colorArray = [
  '#ffcccc',
  '#fee5cb',
  '#ffffcc',
  '#e6ffcc',
  '#ccffcc',
  '#ccffe6',
  '#ccffff',
  '#cce5ff',
  '#fe9898',
  '#ffcc99',
  '#ffff99',
  '#ccff99',
  '#99ff99',
  '#99ffcc',
  '#99ffff',
  '#99ccff',
  '#ff6666',
  '#feb265',
  '#ffff66',
  '#b3ff66',
  '#66ff66',
  '#66ffb3',
  '#66ffff',
  '#64b0fd',
  '#ff3333',
  '#ff9933',
  '#ffff33',
  '#99ff33',
  '#33ff33',
  '#33ff99',
  '#33ffff',
  '#3399ff',
  '#ff0000',
  '#ff8000',
  '#ffff00',
  '#7ffe00',
  '#00ff00',
  '#00ff80',
  '#00ffff',
  '#007fff',
  '#cc0000',
  '#cc6600',
  '#cccc00',
  '#66cc00',
  '#00cc00',
  '#00cc66',
  '#00cbcb',
  '#0066cc',
  '#980000',
  '#994c00',
  '#999900',
  '#4d9900',
  '#009800',
  '#00994d',
  '#009999',
  '#004c99',
  '#660000',
  '#663300',
  '#666600',
  '#336600',
  '#006600',
  '#006633',
  '#006666',
  '#003265',
];