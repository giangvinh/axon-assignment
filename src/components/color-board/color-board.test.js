import React from 'react';
import { shallow } from 'enzyme';
import ColorBoard from './color-board.component';
import {colorArray} from '../../constants/color';

describe('Color board component', () => {

  it('should render color board component', () => {
    expect(shallow(<ColorBoard />)).toMatchSnapshot();
  });

});
