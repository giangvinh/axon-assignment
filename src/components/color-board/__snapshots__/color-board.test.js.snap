// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color board component should render color board component 1`] = `ShallowWrapper {}`;
